cmd/podman/machine/init.go

```diff
@@ -83,7 +83,7 @@ func init() {
 		"reexec", false,
 		"process was rexeced",
 	)
-	flags.MarkHidden("reexec")
+	flags.MarkHidden("reexec") // nolint: errcheck
```

Comment:
Would it be better to do `_ := flags.MarkHidden(...)` instead? Seems a bit more clear

Reply:
agree, please do the same for all other places where this is used.